app/blog/[slug]/page.tsx

```
@@ -0,0 +1,304 @@
+            {/* Table of Contents - Desktop */}
+            <div className="hidden lg:block lg:col-span-1">
+              <div className="sticky top-8">
+                <TableOfContents content={content} />
+              </div>
+            </div>
+
+            {/* Main Content */}
+            <div className="lg:col-span-3">
+              <article className="prose prose-lg prose-blue max-w-none dark:prose-invert">
+                <div 
+                  dangerouslySetInnerHTML={{ __html: content }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)